devdoc/repo_list

```diff
@@ -134,7 +134,6 @@ https://github.com/nix-community/nur-packages-template
 https://github.com/nix-community/nur-search
 https://github.com/nix-community/nur-update
 https://github.com/nix-community/nurl
-https://github.com/nix-community/ofborg-viewer
```

Comment:
I asked samueldr to transfer his repo to nix-community so I could move it to the NixOS org.